src/PrestaShopBundle/Controller/Admin/Sell/Address/AddressController.php

```diff
@@ -394,9 +394,10 @@ public function editAction(int $addressId, Request $request): Response
      *
      * @return Response
      */
-    public function editOrderAction(int $orderId, string $addressType, Request $request): Response
+    public function editOrderAddressAction(int $orderId, string $addressType, Request $request): Response
```

Comment:
But, changing the name of this method can introduce BC ? 🤔

Reply:
No, because this controller is not released yet ;)

Reply:
Sorry I'm in the future  🚀